src/app/content/highlights/components/utils/showConfirmation.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { RawIntlProvider } from 'react-intl';
+import { IntlShape } from 'react-intl';
 import uuid from 'uuid/v4';
-import { AppServices } from '../../../../types';
 import { assertDocument, assertNotNull } from '../../../../utils';
 import ConfirmationModal from '../ConfirmationModal';
 
-export default async(services: AppServices) => {
+export default async(intl: IntlShape) => {
```

Comment:
so i think you can actually just call createIntl and not even bother taking the intl argument, but bartek is gonna need dispatch so the services are gonna end up still in here